chroma-agent/chroma-agent.spec

```diff
@@ -123,6 +124,12 @@ done
 %clean
 rm -rf %{buildroot}
 
+%pre
+if [ $1 -eq 2 ]; then
+  systemctl stop %{name}
+  systemctl disable %{name}
```

Comment:
Why the disable before an upgrade?

How/when does this unit ever get enabled even during an initial install?